docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -1258,6 +1258,44 @@ Renames an existing field. If the field doesn't exist or the new name is already
 }
 --------------------------------------------------
 
+[[script-processor]]
+=== Script Processor
+
+Script Processor allows inline, stored, and file scripts to be executed within Ingest Pipelines.
+
+See <<modules-scripting-using, How to use scripts>> to learn more about writing scripts. Since provided scripts
```

Comment:
Provided scripts makes it sound like the scripts might be provided somehow by the system. So either say "Since the scripts that you provide are potentially compiled..." or remove the word "provided" entirely.